### ISSUE TITLE
Network devices Menu sometimes shows multiple rows

### ISSUE THREAD

**Issue by jdgiddings** [tailscale/tailscale#3894]
### What is the issue?

![image](https://user-images.githubusercontent.com/79322633/153239819-7b099d0f-f4d2-4c63-b77a-81a3e5b96693.png)

This happened while I was messing with ACLs today. It won't go back to normal without a reboot, even if I reverse the ACL changes I made

### Steps to reproduce

Add the following ACL to your Tailnet:

```
{
   "action": "accept",
   "users": ["*"],
   "ports": ["autogroup:self:*"]
},
```
Save your ACLs, now remove that ACL and save again

### Are there any recent changes that introduced the issue?

No

### OS

Windows

### OS version

Windows 11

### Tailscale version

1.20.2

### Bug report

BUG-607682d70a6eaea90b60d6598c3640f2e9dfe61da7c29a20cb5d00856c5b7d6c-20220209160311Z-5ef97ea9e2392edd

**Comment by bradfitz** [comment]
And cc @maisem

**Comment by dblohm7** [comment]
I do see a potential race condition in menu updating code...

**Comment by DentonGentry** [comment]
I'm going to assume that https://github.com/tailscale/corp/pull/3949 (which you won't be able to see, sorry) fixed this in 1.26. Please reopen if it is seen again.
